pkg/oc/cli/admin/release/new.go

```diff
@@ -1466,6 +1470,31 @@ func pruneEmptyDirectories(dir string) error {
 	})
 }
 
+func ensureConsistentCommits(is *imageapi.ImageStream) error {
```

Comment:
This could probably use a doc comment, saying why we want this, e.g.:
```
// ensureConsistentCommits checks that images built from the same git repository are at the same git sha1
```

Reply:
> This could probably use a doc comment...

Added with minor wording tweaks in 3094f69165 -> d1aebc95c1.